steps/telegram-notify/v1.0.0/step.yml

```diff
@@ -0,0 +1,59 @@
+title: telegram_notify
+summary: |
+  Notify group or chat about build status
+description: |
+  Notify group or chat about build status
```

Comment:
Please provide some more details about the step, at least link the used service.